dbos/cli.py

```diff
@@ -131,10 +131,17 @@ def copy_template(src_dir: str, project_name: str) -> None:
         "db_name": db_name,
     }
 
-    copy_template_dir(src_dir, dst_dir, ctx)
-    copy_template_dir(
-        path.join(src_dir, "__package"), path.join(dst_dir, package_name), ctx
-    )
+    if lightweight:
```

Comment:
Print a message saying it's only copying the config file, not the entire template.